features/enlinkd/org.opennms.features.enlinkd.topogen/src/test/java/org/opennms/enlinkd/topogen/Asserts.java

```diff
@@ -0,0 +1,47 @@
+/*******************************************************************************
+ * This file is part of OpenNMS(R).
+ *
+ * Copyright (C) 2019 The OpenNMS Group, Inc.
+ * OpenNMS(R) is Copyright (C) 1999-2019 The OpenNMS Group, Inc.
+ *
+ * OpenNMS(R) is a registered trademark of The OpenNMS Group, Inc.
+ *
+ * OpenNMS(R) is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License,
+ * or (at your option) any later version.
+ *
+ * OpenNMS(R) is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with OpenNMS(R).  If not, see:
+ *      http://www.gnu.org/licenses/
+ *
+ * For more information contact:
+ *     OpenNMS(R) Licensing <license@opennms.org>
+ *     http://www.opennms.org/
+ *     http://www.opennms.com/
+ *******************************************************************************/
+
+package org.opennms.enlinkd.topogen;
+
+import static org.junit.Assert.fail;
+
+public class Asserts {
+
+  /* Taken from a newer jUnit Version */
+  public static <T extends Throwable> void assertThrows(Class<T> expectedType, Runnable executable) {
+    try {
+      executable.run();
+    } catch (Throwable t) {
+      if (expectedType.isInstance(t)) {
+        return; // all good
+      }
+      fail("Expected Exception " + expectedType.getName());
+    }
+  }
```

Comment:
The formatting here seems off.
Can we also point out from which exact version this was copied? 
Can we actually do that without offending any licenses?